src/less/components/code-view.less

```diff
@@ -32,6 +32,10 @@
         border-bottom-right-radius: 4px;
         width: 100%;
 
+        &:focus-visible {
```

Comment:
Maybe add border-radius 2px (or is it 3px that is our standard? check up pls hehe) on tooltip class
Makes the focus look better 👍